app/models/seat.rb

```diff
@@ -0,0 +1,8 @@
+class Seat < ApplicationRecord
+  belongs_to :ticket, optional: true
+  belongs_to :trip, optional: true
```

Comment:
why `optional: true` ?

Reply:
I used `optional: true` because I used `dependent: nullify`.  By using these, the foreign keys will set to NULL when that entry that it is associated with it is removed from the database. If this is not optional, we'll face errors.